Neos.Media/Classes/Domain/ValueObject/Configuration/Variant.php

```diff
@@ -0,0 +1,110 @@
+<?php
+namespace Neos\Media\Domain\ValueObject\Configuration;
+
+/*
+ * This file is part of the Neos.Media package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+final class Variant
+{
+    /**
+     * @var string
+     */
+    private $identifier;
+
+    /**
+     * @var Label
+     */
+    private $label;
+
+    /**
+     * @var string
+     */
+    private $description;
+
+    /**
+     * @var array
+     */
+    private $adjustments = [];
```

Comment:
Seems as $adjustments is never set. Can be removed.

Reply:
It is set in line 64:
```php
            $variant->adjustments[$adjustmentIdentifier] = Adjustment::fromConfiguration($adjustmentIdentifier, $adjustmentConfiguration);
```